### ISSUE TITLE
Blank screen when selecting Community in the sidebar

### ISSUE THREAD

**Issue by bouwew** [hacs/integration#789]
## Version of HACS

0.17.1

## Describe the bug
I get a blank screen when I select the Community item in the sidebar.
When I select reload-screen then the HACS-page shows up.

## Debug log

```text

Add your logs here.
`http://192.168.1.210:8123/hacs_frontend/main.js?v=0.17.1:1:1 Uncaught SyntaxError: Identifier 'e' has already been declared`
```

**Comment by bouwew** [comment]
I also have this error in the log:
`http://192.168.1.210:8123/community_plugin/button-card/button-card.js:1:1 Uncaught SyntaxError: Identifier 'e' has already been declared`

Sometimes my button-cards don't show up, instead I see the red warning-block(s).

**Comment by ludeeus** [comment]
Clear your browser cache

**Comment by bouwew** [comment]
Tried that, seems to work for the button-card but not for HACS, I still see a blank page.

**Comment by bouwew** [comment]
Update: when I refresh the HACS page and then go back to the page with the button-cards, then the red warning-boxes show up and no cards. So they are connected.

**Comment by ludeeus** [comment]
So if you remove button card from your resources and reload, hacs works?

**Comment by bouwew** [comment]
Yes, when I comment out the button-card-url in my `ui-lovelace.yaml` file then the HACS page shows up every time.

**Comment by ludeeus** [comment]
Interesting

**Comment by ludeeus** [comment]
I have tested this with version `3.0.0` of the custom button card, and the `0.17.1` version of HACS, and can not replicate it.

**Comment by Juice2019** [comment]
I have the same issue reaching the new UI, i'm running 0.17.0 (I don't know how to upgrade HACS when I can't access UI). I however is not running custum button card, so I dont think it's a issue with that.

**Comment by ludeeus** [comment]
Add logs here, or create a separate issue.

**Comment by Juice2019** [comment]
Here u go. 
`http://...:8123/hacs_frontend/main.js?v=0.17.0:1:1 Uncaught SyntaxError: Identifier 'r' has already been declared`
I just get an error with 
`http://...:8123/community_plugin/podcast-card/podcast-card.js:1:1 Uncaught SyntaxError: Identifier 'a' has already been declared`
So I did the same, and commented out the podcast-card, as my problem seemed to be the podcast-card.
@ludeeus Do you want me to open a separate issue for the podcast-card?

**Comment by ludeeus** [comment]
this is fine, but update to 0.17.3 and see if that solves it

**Comment by Juice2019** [comment]
That solved it! Thank you. By the way - thanks for the awesome work on the new UI, it looks great! :)

**Comment by bouwew** [comment]
I updated to 17.3, cleared the browser cache etc. Unfortunately the issue I reported is still there.

**Comment by bouwew** [comment]
Maybe this can help: https://www.reddit.com/r/learnjavascript/comments/6swml4/identifier_insert_let_variable_has_already_been/

**Comment by bouwew** [comment]
Managed to update to 0.17.4, the issue is still there.

**Comment by ludeeus** [comment]
Untill I can replicate it, or someone that can replecate fixes it, it will be there.

**Comment by bouwew** [comment]
No worries :)
I know how difficult it can be to fix such issues.

**Comment by ludeeus** [comment]
It's probably easy to fix it, but without knowing what it is it's not possible
